rasa/cdu/command_processor.py

```diff
@@ -358,8 +373,31 @@ def clean_up_commands(
             structlogger.debug(
                 "command_executor.skip_command.slot_already_set", command=command
             )
+        elif isinstance(command, SetSlotCommand) and command.name not in slots_so_far:
+            all_questions = [
+                s for f in all_flows.underlying_flows for s in f.get_question_steps()
```

Comment:
Sorry for para-trooping like this:
I am sure you are already familiar with this and I know that timing is tight, but we should avoid usage of one letter variable. It makes code harder to maintain and read for future developers.

Reply:
weeeeel, i do disagree for for-comprehensions as you usually either need to compromise and split up the comprehension across multiple lines or keep it in a single line with a shorter name.

in this case though, fully agree esp. due to the "double comprehension".

Reply:
`s` easily becomes `step` or `step_item` and `f` becomes `flow` or `flow_item`.
I believe that either comprehension should be a bot easier to read as well.